docker/deb-rpm-s3/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+#
+# Build script for the deb-rpm-s3 docker container.
+#
+
+cd "$(dirname "$0")"
+
+if [ ! -e "elasticsearch.asc" ]; then
+    cat << EOF
+You must place a copy of the Elasticsearch GPG signing key (named
+elasticsearch.asc) into
```

Comment:
As suggested, I will modify the setup to load the GPG key into the container from the volume when the container is launched. This mitigates the risk of an image being pushed that contains the password protected GPG key.